src/licensedcode/data/licenses/ofl-1.1.LICENSE

```diff
@@ -2,7 +2,7 @@
 key: ofl-1.1
 short_name: OFL 1.1
 name: SIL Open Font License 1.1
-category: Permissive
+category: Free Restricted
```

Comment:
@maxhbr @silverhook @DennisClark ping as this relates to https://github.com/nexB/scancode-toolkit/issues/3438
@DennisClark you went with `Free Restricted` over `Copyleft Limited` ... did had a chance to review @silverhook 's comments in https://github.com/nexB/scancode-toolkit/issues/3438#issuecomment-1620113884

> As such, it’s probably best to treat it as Copyleft Limited as it applies only to the font.

> On that note, Ubuntu Font License 1.0, is a (huge) modification of the OFL, falls under the same category (i.e. should be Copyleft Limited).

> In that sense it’s similar to MPL-2.0, which very much is a copyleft license, but limited only to the files that the license is mentioned in.

( FWIW: https://scancode-licensedb.aboutcode.org/ubuntu-font-1.0.html was already "Free Restricted" )

Reply:
Updated in the PR, thanks!

Reply:
Great, @AyanSinhaMahapatra , could you do the same for `ubuntu-font-1.0.LICENSE` too, please?

Reply:
@AyanSinhaMahapatra   please go ahead and update ubuntu-font-1.0 to Copyleft Limited and consider adding this to the Notes: 

From the license text: "The fonts and derivatives, however, cannot be released under any other licence."

Reply:
Done, @DennisClark @silverhook  thanks!